0001-add-metainfo-file.patch

```diff
@@ -0,0 +1,101 @@
+From 5e53001e2b02e345b677fc98331e86701f40d8f0 Mon Sep 17 00:00:00 2001
+From: Martin Marmsoler <martin.marmsoler@gmail.com>
+Date: Wed, 2 Nov 2022 15:03:25 +0100
+Subject: [PATCH 1/2] add metainfo file
+
+---
+ source6/ChartGeany/chartgeany.metainfo.xml | 82 ++++++++++++++++++++++
+ 1 file changed, 82 insertions(+)
+ create mode 100644 source6/ChartGeany/chartgeany.metainfo.xml
+
+diff --git a/source6/ChartGeany/chartgeany.metainfo.xml b/source6/ChartGeany/chartgeany.metainfo.xml
+new file mode 100644
+index 0000000..aa573a1
+--- /dev/null
++++ b/source6/ChartGeany/chartgeany.metainfo.xml
+@@ -0,0 +1,82 @@
++<?xml version="1.0" encoding="UTF-8"?>
++<component type="desktop-application">
++  <id>io.sourceforge.chart_geany.chart_geany</id>
++
++  <name>Chart Geany</name>
++  <summary>Chart Geany is a free cross platform software solution for market technical analysis, charting and a portfolio manager</summary>
++
++  <metadata_license>MIT</metadata_license>
```

Comment:
CC0-1.0 remains prefered for the metadata.

Reply:
changed